Teak/TeakLog.m

```diff
@@ -113,7 +113,16 @@ - (nullable id)initForTeak:(nonnull Teak*)teak withAppId:(nonnull NSString*)appI
 - (void)useSdk:(nonnull NSDictionary*)sdkVersion andXcode:(nonnull NSDictionary*)xcodeVersion {
   self.sdkVersion = sdkVersion;
   self.xcodeVersion = xcodeVersion;
-  [self logEvent:@"sdk_init" level:INFO eventData:nil];
+
+  // Log ISO8601 format timestamp at init
+  NSDateFormatter* formatter = [[NSDateFormatter alloc] init];
+  [formatter setLocale:[NSLocale localeWithLocaleIdentifier:@"en_US_POSIX"]];
+  formatter.timeZone = [NSTimeZone timeZoneWithAbbreviation:@"UTC"];
+  [formatter setDateFormat:@"yyyy-MM-dd'T'HH:mm:ss"];
+  NSString* iso8601String = [formatter stringFromDate:[NSDate date]];
+  
+  NSDictionary* dateTime = @{@"at" : [iso8601String stringByAppendingString:@"Z"]};
```

Comment:
Why not just put the 'Z' in as part of the date format?

Reply:
It doesn't format that properly. It will be something like @"2022-03-08T01:49:57+0000" instead of @"2022-03-08T01:48:31Z"

Reply:
The Z should be in '' like the T, so 'Z'

Reply:
Oh duh. Right.